packages/common/src/visualizations/CartesianChartDataTransformer.ts

```diff
@@ -157,8 +202,12 @@ export type CartesianChartDisplay = {
     yAxis?: {
         label?: string;
         position?: string;
+        format?: Format;
     }[];
-    series?: Record<string, { label?: string; yAxisIndex?: number }>;
+    series?: Record<
+        string,
+        { label?: string; format?: Format; yAxisIndex?: number }
+    >;
```

Comment:
thoughts on each having their own format? 🤔 I think it makes sense

Reply:
Each series? For sure. You might have some percentages and some raw numbers. Or whatever.